src/rgw/rgw_sal.cc

```diff
@@ -337,8 +337,6 @@ DriverManager::Config DriverManager::get_config(bool admin, CephContext* cct)
       if (g_conf().get_val<Option::size_t>("rgw_max_chunk_size") !=
```

Comment:
@cbodley - the check !admin takes care of enabling d3n datacache only for non radosgw-admin operations. For radosgw-admin commands, admin = true. I had missed this check in the d3n filter driver, and the check has been re-instated.